### PR TITLE
test: setup test

### PR DESCRIPTION
**Pull request by felixliao**

### CODE REVIEW

**Comment by felixliao** [comment]
Screenshot 2023-09-22 at 14 59 49

File: apps/web/src/components/chat/chat-input.tsx [hunk 1]
```diff
@@ -43,15 +43,20 @@ const ChatInput = ({
...
-          <Button className=" bg-white" onClick={reload} variant="outline">
+          <Button className="bg-white" onClick={reload} variant="outline">
             <RefreshCw size={16} className="mr-2" />
             Regenerate response
           </Button>
         )}
         {isLoading && (
-          <Button className=" bg-white" onClick={stop} variant="outline">
+          <Button
+            className=" bg-white"
```
Comment (lzl) [inline apps/web/src/components/chat/chat-input.tsx:55]: 这个空格也帮忙顺便给去掉吧。

File: apps/web/src/components/chat/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+
+  beforeAll(() => {
+    vi.mock('@clerk/nextjs', () => ({
+      useUser: () => ({
+        firstName: 'test',
+        lastName: 'user',
+        fullName: 'test user',
+      }),
+    }))
+    vi.mock('@/hooks/use-subscribe', () => ({
```
Comment (lzl) [inline apps/web/src/components/chat/index.test.tsx:65]: TIL 👍

**Review by lzl — APPROVED** [review]
LGTM

File: apps/web/src/components/chat/chat-input.tsx (hunk 1 above)
Comment (felixliao) [reply apps/web/src/components/chat/chat-input.tsx:55]:
```suggestion
            className="bg-white"
```

**Comment by lzl** [comment]
This is a great beginning. When we have enough foundational test code, we can let AI help us write tests.

**Pull request merged**